### DISCUSSION TITLE
Is this suppose to format markdown?

### DISCUSSION THREAD

**Discussion by secondl1ght** [tailwindlabs/tailwindcss#11997] [Help] [answered]
I see in the docs that it is supposed to help with markdown syntax. However I am testing it in an app and when I have text like `**bolded text**` it does not display bold. It also leaves the `**` characters in place.

**Comment by wongjn** [comment, marked as answer]
Assuming this is in reference to the [`@tailwindcss/typography` plugin documentation](https://tailwindcss.com/docs/typography-plugin), this plugin can generate CSS for undeterminable content of an element, but does not render any markdown itself.
